src/main/java/net/snowflake/client/core/CredentialManager.java

```diff
@@ -106,16 +116,26 @@ synchronized void fillCachedCredential(SFLoginInput loginInput, String credType)
     }
 
     if (cred == null) {
-      logger.debug("retrieved %s is null", credType);
+      logger.debug("Retrieved %s is null", credType);
```

Comment:
let's replace `%s` with `{}` in the log message

Reply:
Done